### DISCUSSION TITLE
Log format

### DISCUSSION THREAD

**Discussion by Destroyer** [apache/couchdb#3995] [Q&A] [answered]
I am writing logstash rule for parsing the couchdb logs and I was wondering what some of the columns mean / how to parse it correctly. 
Example:
```
[notice] 2022-04-15T22:00:38.353402Z user@host <0.21856.149> 05f04d6ad8 localhost:5984 127.0.0.1 replicator GET /dbserver-20201020/ 200 ok 1
[info] 2022-04-15T22:00:44.244015Z user@host <0.1763.0> -------- Starting index update for db: shards/40000000-7fffffff/dbserver-20210402.1617383060 idx: _design/test01
```
severity, timestamp, user, <???>  and the rest seems optional

**Comment by big-r81** [comment, marked as answer]
Hi,

the 4th part is the Process-Id (ex. <0.21856.149> of the Erlang process.
